.travis.yml

```diff
@@ -1,28 +1,55 @@
-sudo: false
-
+sudo: required
```

Comment:
Why is `sudo` required?

Reply:
> > We will have one include directory for all header-only classes.
> 
> How will you keep these files in sync with new releases coming from their individual repositories?

We need to get header files from these libraries in current state and put them into our library. Also adapt them to directory structure and namespaces. I don't think that there will be a lot of requests to change them. But if this is true, then we should to talk with developers to continue develop them inside `mapbox-base`

Reply:
Thanks. Fixed